Nodejs/Product/Nodejs/Debugger/NodeDebugProvider.cs

```diff
@@ -117,7 +120,7 @@ private void SetupDebugTargetInfoForWebkitV2Protocol(ref VsDebugTargetInfo vsDeb
                 new JProperty("runtimeExecutable", nodeExe),
                 new JProperty("cwd", cwd),
                 new JProperty("console", "externalTerminal"),
-                new JProperty("diagnosticLogging", NodejsProjectLauncher.CheckEnableDiagnosticLoggingOption()),
+                new JProperty("trace", NodejsProjectLauncher.CheckEnableDiagnosticLoggingOption()),
```

Comment:
This was a bug I missed porting from the ProjectLauncher, i.e. the non AnyCode debugger.